atsign-foundation/atsign.dev-3.0
# First term definition for the AtSign Glossary!

**- What I did**
-Set up repo and Hugo for development on my machine.
- Created a new single page for the privacy definition

**- How I did it**
I pulled the template from archetypes and added my definition draft from the google doc.
**- How to verify it**
Run the changes on a local server to make sure what I added is correct. Should look like this:

![image](https://user-images.githubusercontent.com/34658202/172665087-3fe325d2-6457-40b9-9e7c-fb6f40eb1b8e.png)

**- Description for the changelog**
